### PR TITLE
[EDU-144] Update glossary terms to match new IA

### PR DESCRIPTION
**Pull request by m-hulbert**
## Description

This PR updates several terms in the glossary to be consistent with the changes made to terminology as part of the new IA.

### CODE REVIEW

File: content/root/glossary.textile [hunk 1]
```diff
@@ -201,17 +201,17 @@ Ably Queues provide a service that distributes messages between subscribers, so
...
 See the "queues documentation":/general/queues for further information.
 
-h2(#realtime-sdk). Realtime client library SDK
+h2(#realtime). Realtime interface
 
-The realtime client library SDKs can be used to build applications by providing a simple way to add long-lived realtime connections to Ably. SDKs are available in various languages and platforms, such as JavaScript, Ruby and Android.
+The realtime interface of an Ably SDK is implemented using an Ably-defined protocol, with communication primarily over WebSockets. It enables devices and browsers to maintain a persistent connection to Ably. SDKs are available in various languages and platforms, such as JavaScript, Ruby and Android.
 
-See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+See the "available SDKs":/getting-started/sdks for further information on the realtime interface and the languages an platforms supported by the SDKs.
```
Comment (Srushtika) [inline content/root/glossary.textile:208]:
```suggestion
See the "available SDKs":/getting-started/sdks for further information on the realtime interface and the languages and platforms supported by the SDKs.
```

File: content/root/glossary.textile [hunk 2]
```diff
@@ -201,17 +201,17 @@ Ably Queues provide a service that distributes messages between subscribers, so
...
 
-h2(#rest-sdk). REST client library SDK
+h2(#rest). REST interface
 
-The REST client library SDKs can be used to build applications by providing a simple way to interact with the Ably REST (HTTP) interface. SDKs are available in various languages and platforms, such as JavaScript, Ruby and Android.
+The REST interface of an Ably SDK communicates with the Ably service using the HTTP protocol and is effectively stateless. The REST interface provides a convenient way to access the "REST HTTP API":/api/rest-api and is intended to be used by clients that don't require realtime updates.
 
-See the "SDK downloads page":https://ably.com/download for further information on SDKs and the available languages.
+See the "available SDKs":/getting-started/sdks for further information on the REST interface and the languages an platforms supported by the SDKs.
```
Comment (Srushtika) [inline content/root/glossary.textile:214]:
```suggestion
See the "available SDKs":/getting-started/sdks for further information on the REST interface and the languages and platforms supported by the SDKs.
```

**Review by Srushtika — APPROVED** [review]
Added a couple of suggestions to fix typos, looks good otherwise.

File: content/root/glossary.textile (hunk 1 above)
Comment (m-hulbert) [reply content/root/glossary.textile]: Good spot - fixed! 🙈

**Pull request merged**